Tasks/Maven/maventask.ts

```diff
@@ -7,6 +7,17 @@ import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
 
 import pmd = require('./pmdForMaven');
+import ar = require('./analysisResult');
+
+// Set up for localization
+tl.setResourcePath(path.join( __dirname, 'task.json'));
+
+// Cache build variables - if they are null, we are in a test env and can use test inputs
+var sourcesDir:string = tl.getVariable('build.sourcesDirectory') || tl.getInput('test.sourcesDirectory');
+console.log("sourcesDir = " + sourcesDir);
```

Comment:
Either send this via the verbose stream (console.debug?) or make it more readable i.e. Sources Directory: ...